src/configurations.jl

```diff
@@ -233,78 +310,90 @@ function state_sym(state::AbstractString)
     end
 end
 
-function core_configuration(::Type{O}, element::AbstractString, state::AbstractString) where {O<:AbstractOrbital}
+function core_configuration(::Type{O}, element::AbstractString, state::AbstractString, sorted) where {O<:AbstractOrbital}
     element ∉ keys(noble_gases_configurations[O]) && throw(ArgumentError("Unknown noble gas $(element)"))
     state = state_sym(state == "" ? "c" : state) # By default, we'd like cores to be frozen
     core_config = noble_gases_configurations[O][element]
     Configuration(core_config.orbitals, core_config.occupancy,
-                  [state for o in core_config.orbitals])
+                  [state for o in core_config.orbitals],
+                  sorted=sorted)
 end
 
 function parse_orbital(::Type{O}, orb_str) where {O<:AbstractOrbital}
     m = match(r"^(([0-9]+|.)([a-z]|\[[0-9]+\])[-]{0,1})([0-9]*)([*ci]{0,1})$", orb_str)
     orbital_from_string(O, m[1]) , (m[4] == "") ? 1 : parse(Int, m[4]), state_sym(m[5])
 end
 
-function Base.parse(::Type{Configuration{O}}, conf_str::AbstractString) where {O<:AbstractOrbital}
-    isempty(conf_str) && return Configuration{O}()
+function Base.parse(::Type{Configuration{O}}, conf_str::AbstractString, sorted=false) where {O<:AbstractOrbital}
```

Comment:
Should `sorted` be a keyword argument maybe?

Reply:
b379f57